rack-session.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |spec|
   spec.summary = "A session implementation for Rack."
   spec.license = "MIT"
 
-  spec.files = Dir.glob('{lib}/**/*', File::FNM_DOTMATCH, base: __dir__)
+  spec.files = Dir.glob('lib/**/*.rb')
```

Comment:
I just do this as a matter of standardisation as I've been bitten by both issues you mention in the past, so I'd prefer to leave it as is unless there is a significant advantage to your proposed way. While the code is simpler, it's also more likely to go wrong.

Reply:
The main advantage of this change is it should hopefully pass CI on Ruby 2.4.  I think there is no chance of having dotfiles in `lib`, and the odds that we would ever have a non `.rb` file in `lib` seem quite remote.  I see your approach as much more likely to pick up files accidentally. For example, say someone is using an editor that creates a dot file or non`.rb` file for backup version, and wants to build the gem.  The use of `base` makes no sense, since if you weren't already in the repository directory, the `require_relative` earlier in the gemspec wouldn't work.

You should probably grant @tenderlove and me access to this repository.

Reply:
> For example, say someone is using an editor that creates a dot file or non.rb file for backup version, and wants to build the gem.

The release process ensures that this can never happen as git must report the repository as clean before doing a release.

> The use of base makes no sense, since if you weren't already in the repository directory, the require_relative earlier in the gemspec wouldn't work.

In the past, I've had trouble loading gem specs which have a relative list of files with no concept of a root directory. If you do this, it's almost impossible to load a gem spec out of context. However, I can't recall the exact scenario where it was a problem and I guess it won't be a problem here, so I accept your change. That being said, I may want to revert it in the future if things like external tests become more difficult etc.